index.js

```diff
@@ -1,15 +1,14 @@
-const { generateCorrelationId } = require('@first-lego-league/ms-correlation/lib/correlation-id')
+const { getCorrelationId } = require('@first-lego-league/ms-correlation')
```

Comment:
Why did your change that? This is a client, if it is browser, the domain request will throw an error. We used `generateCorrelationId` in order to see the correlation id from there client (used as header in the mhub message)